src/simulator_core/entities/assets/production_cluster.py

```diff
@@ -111,15 +110,15 @@ def __init__(
         # DemandCluster pressure specifications
         self.pressure_supply = pressure_supply
         self.control_mass_flow = control_mass_flow
-        self._controlled_mass_flow = heat_demand_and_temperature_to_mass_flow(
-            thermal_demand=self.thermal_production_required,
-            temperature_supply=self.temperature_supply,
-            temperature_return=self.temperature_return,
-            pandapipes_net=self.pandapipes_net,
-        )
+#        self._controlled_mass_flow = heat_demand_and_temperature_to_mass_flow(
+#            thermal_demand=self.thermal_production_required,
+#            temperature_supply=self.temperature_supply,
+#            temperature_return=self.temperature_return,
+#            pandapipes_net=self.pandapipes_net,
+#        )
```

Comment:
Why is this removed?  please add a comment with reasoning

Reply:
is now out of the code since it is moved to other function